include/libtorrent/error_code.hpp

```diff
@@ -256,6 +256,8 @@ namespace libtorrent {
 			redirecting,
 			// The HTTP range header is invalid
 			invalid_range,
+			// The HTTP response did have a invalid content length
+			invalid_content_length,
```

Comment:
you can't insert another value here, it will shift all the following error codes one step up and break ABI. Either use `invalid_range` or add another error code at the end of a section (where the next code is assigned a specific value)